requirements.txt

```diff
@@ -37,18 +38,20 @@ kinto-ldap==0.3.1
 kinto-signer==2.1.1
 ldappool==2.1.0
 logging-color-formatter==1.0.2
-lxml==4.1.0
+lxml==4.1.1
 MarkupSafe==1.0
 mohawk==0.3.4
 mozilla-cloud-services-logger==1.0.1
```

Comment:
This should not be necessary, but let's leave it until the log configs are changed